helm-chart/renku-gateway/values.yaml

```diff
@@ -23,6 +23,14 @@ global:
     ## set by parent chart.
     domain: example.local
 
+    ## The name of the Keycloak Realm to be used is set to 'Renku'
```

Comment:
I find this wording a bit confusing -- how about:

"Set the Keycloak Realm name used by Renku here - the default value is "Renku". You may change this if you are using an external Keycloak instance and want to use an existing realm. If Keycloak is deployed as a part of Renku DO NOT change this value."